src/tactic/ui/startup/project_config_wdg.py

```diff
@@ -212,47 +212,30 @@ def get_panels(my):
 
         show_security = my.kwargs.get("show_security")
 
-
-        user_panel = DivWdg()
-        user_panel.add_style("padding-top: 3px")
-        user_panel.add_style("overflow-y: auto")
-        user_panel.add( UserPanelWdg(show_security=show_security) )
-        user_panel.add_style("min-height: 100px")
-        user_panel.add_style("height: 300px")
-        user_panel.add_class("spt_resizable")
-
-        panel = {
-            'widget': user_panel,
-            'title': 'User List',
-        }
-        panels.append(panel)
-
-
         from tactic.ui.container import TabWdg
         config_xml = []
 
         config_xml.append('''
         <config>
-        <tab>
         ''')
 
-        """
-        <element name="Help">
-            <display class='tactic.ui.app.HelpContentWideWdg'>
-              <alias>main</alias>
-              <width>1000px</width>
-            </display>
-        </element>''')
-        """
```

Comment:
I'm not sure why this HelpContentWideWdg was here; removing it did nothing; not sure if this is a bug or if we should leave it out